specifications/transactions/README.md

```diff
@@ -57,7 +57,7 @@ There are two cases for refunds, those initiated by the recipient and those that
 
 The rest of this specification contains the following components:
 * [On-Chain Data and Transactions](onchain) including metadata and blockchain data
-* Off-Chain Identifiers including Account Addresses, Subaddresses, and PaymentURIs
+* [Off-Chain Identifiers](offchain-identity) including Account Addresses, Subaddresses, and PaymentURIs
```

Comment:
this is not quite right as account addresses belong to on-chain data and payment uris / intent identifiers need a revamp that @sunmilee and I working on... will update after some reviews...

this should just say subaddresses for now